mobile/ios/scripts/build_arch.sh

```diff
@@ -38,15 +46,15 @@ export CXXFLAGS="-isysroot ${DEVELOPER}/Platforms/${PLATFORM}.platform/Developer
                 --prefix=/ \
                 $EXTRA_CONFIG
     make -j4 V=0
-    make install V=0 DESTDIR=$REPOROOT/build/${PLATFORM}/${ARCH}
+    make install V=0 DESTDIR=$BUILDDIR/build/${PLATFORM}/${ARCH}
```

Comment:
From what I learned [in the Android case](https://github.com/measurement-kit/measurement-kit/blob/master/mobile/android/scripts/build_target.sh#L90), a good way to install MeasurementKit is the following:

1. make install-strip $OPTIONS
2. rm -rf of the installed but unneeded headers
3. make install-data-am
4. rm -rf of more installed stuff we don't need

This allows to install stripped libraries (that consume less disk space) and only install measurement-kit headers since for some time now all the other headers of dependencies are not needed anymore.